app/controllers/idv/review_controller.rb

```diff
@@ -52,6 +53,15 @@ def create
 
     private
 
+    def step_indicator_steps
+      steps = Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS
+      return steps if idv_session.address_verification_mechanism != 'gpo'
+      steps.map do |step|
+        step = { name: step, status: :pending } if step == :verify_phone_or_address
+        step
```

Comment:
I don't like reassigning the loop variable to a different type here, could we do:

```suggestion
        if step == :verify_phone_or_address
          { name: step, status: :pending }
        else
          step
        end
```

Reply:
Curious, is the following something you'd think is reasonable, or is there otherwise a common syntax I'm overlooking for returning early from a map block? Or would the `if` / `else` still tend to be most common / preferred?

```suggestion
        next { name: step, status: :pending } if step == :verify_phone_or_address
        step
```

Reply:
personally I like the big if/else blocks but I know rubocop favors the one-liners and trailing ifs

I don't think this is that much better or worse, happy to trust your call on it

Reply:
I took care of this indirectly through d37a4362f, which also addresses your other review point.

Not sure if the concern with reassignment is specifically about changing types, or just reassignment of the block argument generally. If it's only the former, it could still be simplified a bit more now that we're at least using hashes consistently.

```rb
        step = step.merge({ status: :pending }) if step[:name] == :verify_phone_or_address
        step
```

But I went with the `if` / `else` as the safer bet.

Reply:
Personally I don't like both, reassigning the block argument and especially changing the type to something very different